package.json

```diff
@@ -6,6 +6,9 @@
   "bin": {
     "addons-linter": "bin/addons-linter"
   },
+  "engines": {
+    "node": ">=10.15"
+  },
```

Comment:
Let's not enforce this requirement. We are advertising compatibility but as of right now we don't break compatibility with earlier nodejs versions so we shouldn't force users to do so if they dare to use node 8 with the linter.